package.json

```diff
@@ -16,16 +16,16 @@
     "bugs": {
         "url": "https://github.com/eventbrite/eventbrite-sdk-javascript/issues"
     },
-    "homepage":
-        "https://github.com/eventbrite/eventbrite-sdk-javascript#readme",
+    "homepage": "https://github.com/eventbrite/eventbrite-sdk-javascript#readme",
     "license": "MIT",
     "scripts": {
-        "check:static": "npm-run-all --parallel lint 'tsc --noEmit'",
+        "check:static": "npm-run-all --parallel lint tsc",
```

Comment:
don't we want `--noEmit` still for this for static checking?

Reply:
its in the tsconfig now